src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -304,6 +304,39 @@ public void execute(Boolean succeeded)
       return deps.toArray(new Dependency[deps.size()]);
    }
    
+   public void withDataImportSAV(String userAction, final Command command)
+   {
+     withDependencies(
+        "Preparing Import from SAV",
+        userAction, 
+        dataImportSavDependenciesArray(), 
+        false,
+        new CommandWithArg<Boolean>()
+        {
+           @Override
+           public void execute(Boolean succeeded)
+           {
+              if (succeeded)
+                 command.execute();
+           }
+        }
+     );
+   }
+   
+   private ArrayList<Dependency> dataImportSavDependencies()
+   {
+      ArrayList<Dependency> deps = new ArrayList<Dependency>();
+      deps.add(Dependency.cranPackage("haven", "0.2.0"));
```

Comment:
Note that the "haven" package requires R version 3.1 and the readr package requires R version 3.0. Technically RStudio supports as far back as R 2.11! That said, I don't think we need to make UI conditional on R >= 3 as a ton of our other UI including Shiny, R Markdown, Rcpp, devtools, etc. are all already predicated on R >= 3. I think we should however check for R >= 3.1 on the server and pass that up through `SessionInfo`, then check it in Application.initializeWorkbench and disable the haven import commands if we don't have R 3.1 or greater.